Upgrading to SPDK 24.01 (#56)
openebs/spdk-rs: README.md

* ISAL is now mandatory
* When upgrading from previous versions, clean the SPDK directory
  before building (e.g. git clean).

> * An `AS` shell var must be set, either `yasm` or `nasm`.

Reviewer: Only nasm works now right?

```suggestion
* An `AS` shell var must be set, with `nasm`.
```

Author: I'm going to rewrite the README.
